integration_tests/tests/vault/set_fees.rs

```diff
@@ -0,0 +1,173 @@
+            .await
+            .unwrap();
+
+        let vault = vault_program_client.get_vault(&vault_pubkey).await.unwrap();
+        let config_pubkey = Config::find_program_address(&jito_vault_program::id()).0;
+        let config = vault_program_client
+            .get_config(&config_pubkey)
+            .await
+            .unwrap();
+
+        assert_eq!(vault.deposit_fee_bps(), deposit_fee_bps);
+        assert_eq!(vault.withdrawal_fee_bps(), withdrawal_fee_bps);
+
+        // Test Setting the Fees
+        fixture
+            .warp_slot_incremental(config.epoch_length * 2)
+            .await
+            .unwrap();
+
+        let deposit_fee_bps = 199;
+        let withdrawal_fee_bps = 200;
+
+        vault_program_client
+            .set_fees(
+                &config_pubkey,
+                &vault_pubkey,
+                &vault_admin,
+                deposit_fee_bps,
+                withdrawal_fee_bps,
+            )
+            .await
+            .unwrap();
+
+        let vault = vault_program_client.get_vault(&vault_pubkey).await.unwrap();
+        assert_eq!(vault.deposit_fee_bps(), deposit_fee_bps);
+        assert_eq!(vault.withdrawal_fee_bps(), withdrawal_fee_bps);
+
+        // Should not be able to set fees again
+        let result = vault_program_client
+            .set_fees(
+                &config_pubkey,
+                &vault_pubkey,
+                &vault_admin,
+                deposit_fee_bps,
+                withdrawal_fee_bps,
+            )
+            .await;
+
+        assert!(result.is_err());
+
+        fixture
+            .warp_slot_incremental(config.epoch_length * 2)
+            .await
+            .unwrap();
+
+        let deposit_fee_bps = 299;
+        let withdrawal_fee_bps = 300;
+
+        vault_program_client
+            .set_fees(
+                &config_pubkey,
+                &vault_pubkey,
+                &vault_admin,
+                deposit_fee_bps,
+                withdrawal_fee_bps,
+            )
+            .await
+            .unwrap();
+
+        let vault = vault_program_client.get_vault(&vault_pubkey).await.unwrap();
+        assert_eq!(vault.deposit_fee_bps(), deposit_fee_bps);
+        assert_eq!(vault.withdrawal_fee_bps(), withdrawal_fee_bps);
+
+        // Should not be able to set fees with wrong signer
+        let vault_admin = Keypair::new();
+        vault_program_client
+            ._airdrop(&vault_admin.pubkey(), 10.0)
+            .await
+            .unwrap();
+
+        fixture
+            .warp_slot_incremental(config.epoch_length * 2)
+            .await
+            .unwrap();
+
+        let result = vault_program_client
+            .set_fees(
+                &config_pubkey,
+                &vault_pubkey,
+                &vault_admin,
+                deposit_fee_bps,
+                withdrawal_fee_bps,
+            )
+            .await;
+
+        assert!(result.is_err());
```

Comment:
please check error type. can look at some other tests for ensuring the error type is correct

Reply:
(found a few alreadyprocessed/test bugs when i started checking for the right errors, so wasn't actually testing the thing i wanted to)

Reply:
Done